01-basics/02-VueCalculator/index.html

```diff
@@ -31,23 +31,23 @@
     <div id="app" class="container">
       <div class="row">
         <div class="col">
-          <input type="number" />
+          <input type="number" v-model="firstOperand" />
         </div>
 
         <div class="col" style="display: flex; flex-direction: column; font-family: emoji">
-          <label><input type="radio" name="operator" value="sum" /> ➕</label>
-          <label><input type="radio" name="operator" value="subtract" /> ➖</label>
-          <label><input type="radio" name="operator" value="multiply" /> ✖</label>
-          <label><input type="radio" name="operator" value="divide" /> ➗</label>
+          <label><input type="radio" name="operator" value="sum" @change="handleOperatorChange" /> ➕</label>
+          <label><input type="radio" name="operator" value="subtract" @change="handleOperatorChange" /> ➖</label>
+          <label><input type="radio" name="operator" value="multiply" @change="handleOperatorChange"/> ✖</label>
+          <label><input type="radio" name="operator" value="divide" @change="handleOperatorChange" /> ➗</label>
```

Comment:
All possible parts of the visible state of an application must be defined as the application state and be managed by the application. 

Here:
1. The state of radio buttons is not bound to the application state. Changing `operation` doesn't lead to changing of a visually checked radio button. `checked` state of every button must be set with `v-bind`.
2. `v-bind` + `v-on` (`@change`) can be shortened with two-way binding directive `v-model`.

Reply:
Fixed with v-model